paasta_tools/marathon_tools.py

```diff
@@ -255,7 +256,7 @@ def get_backoff_seconds(self):
         if instances == 0:
             return 1
         else:
-            return float(10.0 / instances)
```

Comment:
Alright, as long as we are aware that this will bounce everything.

Also I'm assuming that the existing versions of marathon we have in production can handle ints (seems like they should, line 257 is an int)